src/timeline/js/functions.js

```diff
@@ -247,7 +247,12 @@ function setBoundingBox(scope, item, item_type="clip") {
 
     // Compute width from `time` duration (for more accuracy). Getting the width from
     // JQuery is not accurate, and is occasionally rounded.
```

Comment:
item_object can also be NULL, and we need to fallback to the JQuery element width (pixels)